add InvalidObject (#45)
patternfly/react-component-groups: packages/module/patternfly-docs/content/extensions/component-groups/examples/InvalidObject/InvalidObject.md

---
# Sidenav top-level section
# should be the same for all markdown files
section: extensions
subsection: Component groups
# Sidenav secondary level section
# should be the same for all markdown files

> id: InvalidObject

Reviewer: ```suggestion
id: Invalid object
```

# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
source: react
# If you use typescript, the name of the interface to display props for
# These are found through the sourceProps function provided in patternfly-docs.source.js
propComponents: ['InvalidObject']
---

import InvalidObject from '@patternfly/react-component-groups/dist/dynamic/InvalidObject';

> This component can be used to display when an error occurs. It shows an error and a button that allows you to go back to the homepage. 

Reviewer 2: We need @edonehoo  to review the documentation part

> This component can be used to display when an error occurs. It shows an error and a button that allows you to go back to the homepage. 

Reviewer: ```suggestion
The **invalid object** component can be used to display an error message and "return to homepage" button when an error occurs.
```

> ## Component usage
> 
> ### Example
> 

Reviewer: ```suggestion
## Examples

### Basic invalid object 

A basic invalid object should have a `<Title>` that gives users context on the error and gives them options to navigate the error. These options should include a `<Button>` that redirects users to another page, such as a homepage.

```

This is what I'm gathering from the tsx? lmk if this is wrong

Author: I don't think this is right. Title and Button are not options. They're both set in stone as to what they say and do if that makes sense. it just throws an error with a link to the home page.

Reviewer 2: Correct @edonehoo the only option is isBeta prop that will /beta on to the home page when they navigate back by clicking the link/button.

Reviewer: @InsaneZein @dlabaj I updated my above suggestion to remove the former phrasing ^^

Author: looks good. Thank you!

---
# Sidenav top-level section
# should be the same for all markdown files
section: extensions
subsection: Component groups
# Sidenav secondary level section
# should be the same for all markdown files
id: Invalid object
# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
source: react
# If you use typescript, the name of the interface to display props for
# These are found through the sourceProps function provided in patternfly-docs.source.js
propComponents: ['InvalidObject']
---

import InvalidObject from '@patternfly/react-component-groups/dist/dynamic/InvalidObject';

> 

Reviewer 3: ```suggestion
```

The **invalid object** component can be used to display an error message and "return to homepage" button when an error occurs.

> 
> 

Reviewer 3: ```suggestion
```
